src/views/Post.vue

```diff
@@ -50,6 +54,7 @@ export default {
       /* Then fetch data from DB. */
       this.refresh()
     },
+
     // @TODO: 매번 refresh 하는게 최선인지는 좀 생각해 봐야할듯
     async refresh () {
       this.post = await fetchPost({ postId: this.postId })
```

Comment:
Does comment write/update/delete API returns updated comments array or object? If it does, please update the comment object(or an array) with that return value.

Or if retrieve comments API is available, call it and update the comment object.

Reply:
Oh I think I've forgot updating it. Thank you for pointing out!